frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadFileForm.component.tsx

```diff
@@ -44,6 +46,13 @@ export const UploadFileForm = ({ openState, onClickClose }: IUploadFileForm): JS
 		>
 			<Container>
 				<input {...register('firstName')} placeholder="First name" />
+				<DragAndDrop
+					message={formatMessage(
+						{ id: 'containers.upload.message', defaultMessage: 'Supported file formats: IFC, RVT, DGN, FBX, OBJ and { link }' },
+						{ link: <a href="https://help.3drepo.io/en/articles/4798885-supported-file-formats" target="_blank" rel="noreferrer">more</a> },
```

Comment:
Would the link also not change depending on the language though?

Reply:
Yes, I suppose so. But the help articles for now are only in english so...🤷‍♂️

Reply:
If we ever did support multlanguage in the help section it would be by replacing the langage identifier in the url. eg.

https://help.3drepo.io/en/collections/2615586-3d-repo-basics
https://help.3drepo.io/es/collections/2615586-3d-repo-basics
https://help.3drepo.io/fr/collections/2615586-3d-repo-basics

We probably won't do that for a long time though